### PR TITLE
rfc manage remote_identity

### PR DESCRIPTION
**Pull request by iamdey**

### CODE REVIEW

File: _posts/rfc/2016-08-31-remote-identity.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+{
+  "remote_identities": [
+    {
+      "remote_identity_id": "0ba2e346-e4f8-4c45-9adc-eeb1d42f07e2",
+      "is_fetching": true,
+      "connected": false,
+      "last_connection": "2016-08-31T10:15:39+0200",
+      "connection_required": true,
+      "cancel_fetch_required": false,
+      "identity_type": "email",
```
Comment (gdchamal) [inline _posts/rfc/2016-08-31-remote-identity.md]: missing identity_id

File: _posts/rfc/2016-08-31-remote-identity.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+ * is_fetching: calculated by the API, a fetching operation is running
+ * connected: calculated by the API, the result state of the last connection attempt
+ * connection_required: user asked to connect
+ * cancel_fetch_required: user asked to cancel fetching
+ * identity_type: determine witch contact sub-object to use, can be one of ['email', 'im', 'social_identitiy']
+ * identity_id: the sub-object id
+ * params: an "free" with the properties required by the third party plateform
+ * custom param entries:
+   * mail_protocol: standard `IMAP` or `POP` are allowed; `gmail` can be also a valid entry
```
Comment (gdchamal) [inline _posts/rfc/2016-08-31-remote-identity.md]: gmail will be a property of the email contact, not for this remote identity

Comment (iamdey) [inline _posts/rfc/2016-08-31-remote-identity.md]: I'm not sure about that : 
* a custom email can be managed by gmail like johndoe@compagny.tld
* we don't store the provider of the email in contact sub-object

File: _posts/rfc/2016-08-31-remote-identity.md (hunk 1 above)
Comment (iamdey) [inline _posts/rfc/2016-08-31-remote-identity.md]: right

File: _posts/rfc/2016-08-31-remote-identity.md (hunk 2 above)
Comment (gdchamal) [inline _posts/rfc/2016-08-31-remote-identity.md]: I prefer to add a provider parameter and not use gmail (at this time) as a possible mail_protocol value

File: _posts/rfc/2016-08-31-remote-identity.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+ * cancel_fetch_required: user asked to cancel fetching
+ * identity_type: determine witch contact sub-object to use, can be one of ['email', 'im',
+ 'social_identitiy']
+ * identity_id: the sub-object id
+ * params: an "free" with the properties required by the third party plateform, the sub-object of
+ contact will determine what kind of provider will be used
+
+## Specific notes
+
+**Gmail** is a bit special because it is a mail that can be connected by IMAP, POP and its API.
```
Comment (iamdey) [inline _posts/rfc/2016-08-31-remote-identity.md]: WDYT @gdchamal ?

Comment (gdchamal) [inline _posts/rfc/2016-08-31-remote-identity.md]: yes sound better

**Comment by gdchamal** [comment]
LGTM

**Pull request merged**